apps/docs/tests/e2e/overviewCheck.cy.ts

```diff
@@ -0,0 +1,22 @@
+import { OverviewVisibleObject } from '../support/pageObject/OverviewVisible.PageObject';
+
+const overviewVisible = () => new OverviewVisibleObject();
+
+describe('Blocks and Components overview check', () => {
+  it('Vue', () => {
+    cy.visit('/#vue');
+    cy.contains('Install for Vue').should('be.visible');
+    cy.get(`h2[id="base-components"]`).should('contain.text', 'Base Components ');
```

Comment:
If you query by `id` you can remove `h2` selector. It should be better, because `<h2>` may change, but `id` will stay the same. Do the same in the rest of queries.
```suggestion
    cy.get('#base-components').should('contain.text', 'Base Components ');
```